.github/workflows/lint.yml

```
@@ -0,0 +1,26 @@
+name: Lint
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js 18.x # TODO: Update to 20 once TS changes are in.
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

```suggestion
      - name: Checkout Repo
        uses: actions/checkout@v4

      - name: Setup Node.js
        uses: actions/setup-node@v4
        with:
          node-version-file: ".nvmrc"
```

This uses the nvmrc so we don't have to worry about mismatched versions

---

Amazing, thank you!